zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -1,32 +1,108 @@
+//! Prints Zebra checkpoints as "height hash" output lines.
+//!
+//! Get all the blocks up to network current tip and print the ones that are
+//! checkpoints according to rules.
+//!
+//! Usage: zebra-checkpoints [network] --cli <cli-path>
+//! Where network can be `--regtest`, `mainnet` or `--testnet`.
+//! `--cli` is the path of the zcash-cli binary as a string.
+//! If no network is specified program will use the mainnet.
+//!
+//! zebra-consensus accepts an ordered list of checkpoints, starting with the
+//! genesis block. Checkpoint heights can be chosen arbitrarily.
+
 #![allow(clippy::try_err)]
-use color_eyre::eyre::{eyre, Result};
+use color_eyre::eyre::Result;
+use serde_json::Value;
+use std::process::Stdio;
 use structopt::StructOpt;
 
 mod args;
 
+/// We limit the maximum number of blocks in each checkpoint. Each block uses a
+/// constant amount of memory for the supporting data structures and futures.
+const MAX_CHECKPOINT_HEIGHT_GAP: usize = zebrad::commands::LOOKAHEAD_LIMIT / 2;
```

Comment:
I'm not sure we want to depend on zebrad here.

Let's put a DEFAULT_MAX_CHECKPOINT_HEIGHT_GAP in zebra-consensus, and then make zebra-utils and zebrad depend on it.

Reply:
Done at https://github.com/ZcashFoundation/zebra/pull/740/commits/6dbf24edaf6accb80dcc24fef56d4611bab224c3